app/src/app/Marketplace.jsx

```diff
@@ -16,6 +16,7 @@ import routes from '$routes'
 const Route = withErrorBoundary(ErrorPage)(RouterRoute)
 
 const MarketplaceRouter = () => ([
+    <Route exact path={routes.root()} component={Products} key="root" />, // edge case for localhost
```

Comment:
This was needed when logging out and returning to product listing, without this it displays an error (before part of the editor router for some reason)